src/opendr/perception/multimodal_human_centric/rgbd_hand_gesture_learner/rgbd_hand_gesture_learner.py

```diff
@@ -0,0 +1,506 @@
+        assert os.path.exists(metadata_file),\
+            'Metadata file ("metadata.json") does not exist under the given path "{}"'.format(path)
+
+        fid = open(metadata_file, 'r')
+        metadata = json.load(fid)
+        fid.close()
+
+        if metadata['is_builtin_architecture'] and metadata['architecture'] != self.architecture:
+            msg = 'The architecture specification of saved model does not match current model instance\n'
+            msg += '`architecture` parameter of the saved model is: "{}"'.format(metadata['architecture'])
+            msg += '`architecture` parameter of current model instance is: "{}"'.format(self.architecture)
+            raise ValueError(msg)
+
+        model_weight_file = os.path.join(path, metadata['model_paths'][0])
+        assert os.path.exists(model_weight_file),\
+            'Model weights "{}" does not exist'.format(model_weight_file)
+
+        self.model.cpu()
+        self.model.load_state_dict(torch.load(model_weight_file, map_location=torch.device('cpu')))
+
+        if verbose:
+            print('Pretrained model is loaded successfully')
+
+    def download(self, path):
+        """
+        This function is used to download a pretrained model for the hand gesture recognition task
+        Calling load(path) after this function will load the downloaded model weights
+
+        :param path: path to the saved location. Under this path `model_weights.pt` and `metadata.json`
+                     will be downloaded so different paths for different models should be given to avoid
+                     overwriting previously downloaded model
+        :type path: str
+        """
+
+        if not os.path.exists(path):
+            os.makedirs(path, exist_ok=True)
+
+        if isinstance(self.architecture, str) and self.architecture in PRETRAINED_MODEL:
+            assert self.n_class == 16,\
+                'For pretrained hand gesture model, `n_class` must be 16'
+
+            server_url = os.path.join(OPENDR_SERVER_URL,
+                                      'perception',
+                                      'multimodal_human_centric',
+                                      'rgbd_hand_gesture_learner')
+
+            model_name = '{}_{}'.format('hand_gesture', self.architecture)
+
+            metadata_url = os.path.join(server_url, '{}.json'.format(model_name))
+            metadata_file = os.path.join(path, 'metadata.json')
+            urlretrieve(metadata_url, metadata_file)
+
+            weights_url = os.path.join(server_url, '{}.pt'.format(model_name))
+            weights_file = os.path.join(path, 'model_weights.pt')
+            urlretrieve(weights_url, weights_file)
+            print('Pretrained model downloaded to the following directory\n{}'.format(path))
+        else:
+            raise UserWarning('No pretrained model for architecture "{}"'.format(self.architecture))
+
+    def optimize(self):
+        pass
```

Comment:
it could maybe be interesting to raise an error as done in other methods
```suggestion
        raise NotImplementedError
```